### PR TITLE
Support both uint8 and float32 in LiteDualNet.

### PR DESCRIPTION
**Pull request by chsigg**
This is in preparation of adding a test that verifies that the different backends produce the
same result.

### CODE REVIEW

File: cc/dual_net/lite_dual_net.cc [hunk 1]
```diff
@@ -51,8 +58,12 @@ class LiteDualNet : public DualNet {
   std::string graph_path_;
 };
 
-LiteDualNet::LiteDualNet(const std::string& graph_path)
+minigo::LiteDualNet::LiteDualNet(std::string graph_path)
     : graph_path_(graph_path) {
```
[the remark this answers is not in the document]
Comment (chsigg) [reply cc/dual_net/lite_dual_net.cc]: I still need the graph_path below to potentially add the file extension.

File: cc/dual_net/lite_dual_net.cc [hunk 2]
```diff
@@ -123,33 +176,26 @@ void LiteDualNet::RunMany(std::vector<const BoardFeatures*> features,
...
   // TODO(tommadams): Make BoardFeatures a uint8_t array and memcpy here.
-  auto* data = input_->data.uint8;
+  const auto input_params = input_->params;
   for (size_t j = 0; j < features.size(); ++j) {
     const auto& board = *features[j];
     for (size_t i = 0; i < board.size(); ++i) {
       // TODO(csigg): Apply dequantization parameters?
-      data[j * kNumStoneFeatures + i] = static_cast<uint8_t>(board[i]);
+      feature_data[j * kNumStoneFeatures + i] =
+          Convert<T>(input_params, board[i]);
```
Comment (chsigg) [inline cc/dual_net/lite_dual_net.cc]: Unlike before, this applies the params to the input as well. Could you please confirm that this is what we would want?

File: cc/dual_net/lite_dual_net.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (tommadams) [reply cc/dual_net/lite_dual_net.cc]: But you have `graph_path_`...

I'd prefer that `graph_path_` was set to the actual path loaded anyway.

File: cc/dual_net/lite_dual_net.cc (hunk 2 above)
Comment (tommadams) [reply cc/dual_net/lite_dual_net.cc]: I think so. I checked one of our tflite models and it had `scale == 1` and `zero_point == 0`, so the conversion is a no-op in our case. Better safe than sorry though :)

**Pull request closed without merging**